pkg/tools/cloudformationguard/testdata/cloudformation-guard/.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
For dev reasons I merged the official repo as a subtree so that there is testdata. Can drop this once we have our own test corpus.

Reply:
Will add the test corpus to the GH action prior to merge (and drop from this PR)